test/metabase/query_processor/middleware/expand_macros_test.clj

```diff
@@ -11,15 +13,30 @@
    [metabase.test :as mt]))
 
 (defn- mbql-query [inner-query]
-  {:database 1, :type :query, :query (merge {:source-table 1}
-                                            inner-query)})
+  {:database (meta/id)
+   :type     :query
+   :query    (merge {:source-table 1}
+                    inner-query)})
+
+(defn- expand-macros
```

Comment:
I didn't want to rewrite all of these tests (yet), and I wanted to prove that the new simpler middleware is working the same as the old hairy middleware, so I put a bit of magic in to convert to pMBQL and back to legacy so the old tests could otherwise remain unchanged.